frontend/src/pages/components/displayAlbums.js

```diff
@@ -0,0 +1,99 @@
+// +-------------------+----------------------------------------------------------------------
+// | IMPORTS           |
+// +-------------------+
+import React, { useState, useEffect, Component, useContext } from "react";
+import { Link } from "react-router-dom";
+import { Button, Card, Carousel, Col, Row } from "react-bootstrap";
+import MISTImage from "./MISTImageGallery"
+import {UserContext} from './Contexts/UserContext';
+
+import "../../design/styleSheets/profile.css";
+import "../../design/styleSheets/generalStyles.css";
+import "bootstrap/dist/css/bootstrap.css";
+/* icons */
+import {
+  MoreIcon,
+  PrivacyIcon,
+} from "./icons.js"
+
+// +-------------------+----------------------------------------------------------------------
+// | displayAlbums.js  |
+// +-------------------+
+/** Displays albums view when Albums is called
+ */
+function DisplayAlbums() {
+    const [images, setImages] = useState("");
+    const [modalShow, setModalShow] = React.useState(false);
+    const user = useContext(UserContext);
+    return (
+        <Row style={{ justifyContent: "space-between" }}>
+          {console.log(user)}
+          {user.albums.map((album, index) => (
+            <Card
+              style={{ width: "31%", marginTop: "1em", marginBottom: "1em" }}s
+            >
+              <Card.Header>
+                {/* Privacy Icon + Title + MoreIcon */}
+                <Card.Title style={{ margin: "auto" }} className='linkItem'>
+                  <Row style={{ justifyContent: "space-between", alignItems: "center" }}>
+                    <PrivacyIcon />
+                    <Col>
+                      <Link to={{ pathname: `/profile/${album._id}` }} className="link" style={{ color: "black" }}>
+                        <p>{album.name}</p>
+                      </Link>
+                    </Col>
+                    <MoreIcon style={{backgroundColor: "red"}}/>
+                  </Row>
+                </Card.Title>
+  
+                {/* Carousel + Description + date */}
+                <Card.Body style={{ justifyContent: "space-between" }}>
+                  <AlbumCarousel album={album} />
+                  <p>{album.caption}</p>
+                  <p>{new Date(parseInt(album.updatedAt)).toString()}</p>
```

Comment:
I realize I said toString, but toDateString looks nicer (it doesn't have the numbers at the end)

Reply:
What numbers? Do you mean the time?